### DISCUSSION TITLE
[Enhancement] [select] el-select多选时能不能设置最少选几个

### DISCUSSION THREAD

**Discussion by daisyQAQ** [element-plus/element-plus#13862] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `select`

## Description

el-select多选时有最多设置几个选项的功能，能不能增加一个最少选几个功能

### Additional comments

_(empty)_

**Comment by kooriookami** [comment]
假如最少选3个，你连第一个都选不上。
